lib/builder/factory.js

```diff
@@ -74,11 +74,11 @@ factory.getExecutablePath = function (platform) {
     if (platform === 'aix') {
         ret = '/usr/sbin/ping';
     } else if (factory.isLinux(platform)) {
-        ret = '/bin/ping';
+        ret = v6 ? '/sbin/ping6' : '/bin/ping';
     } else if (factory.isWindow(platform)) {
         ret = process.env.SystemRoot + '/system32/ping.exe';
```

Comment:
Yeah, we probably should. I can test on Windows. I'll make the code force either IPv4 or IPv6.